edi_oca/views/edi_exchange_record_views.xml

```diff
@@ -386,4 +386,15 @@
         <field name="view_mode">tree</field>
         <field name="view_id" ref="edi_exchange_record_view_tree" />
     </record>
+    <record model="ir.actions.server" id="action_retry_edi_exchange_record">
```

Comment:
this is not a view, can you please move it to `data`

Reply:
Thank @simahawk , I updated the PR